### YamlMime:ManagedReference
items:
- uid: System.IdentityModel.SignatureVerificationFailedException
  id: SignatureVerificationFailedException
  children:
  - System.IdentityModel.SignatureVerificationFailedException.#ctor
  - System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String)
  - System.IdentityModel.SignatureVerificationFailedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: SignatureVerificationFailedException
  nameWithType: SignatureVerificationFailedException
  fullName: System.IdentityModel.SignatureVerificationFailedException
  type: Class
  summary: "De uitzondering die wordt gegenereerd wanneer een fout optreedt tijdens het verwerken van een handtekening"
  syntax:
    content: 'public class SignatureVerificationFailedException : System.IdentityModel.Tokens.SecurityTokenException'
  inheritance:
  - System.Exception
  - System.SystemException
  - System.IdentityModel.Tokens.SecurityTokenException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor
  id: '#ctor'
  parent: System.IdentityModel.SignatureVerificationFailedException
  langs:
  - csharp
  name: SignatureVerificationFailedException()
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException()
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException()
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.IdentityModel.SignatureVerificationFailedException&quot;> </xref> klasse."
  remarks: "Deze constructor initialiseert de <xref:System.Exception.Message%2A>eigenschap van het nieuwe exemplaar voor een systeem geleverd bericht met een beschrijving van de fout.</xref:System.Exception.Message%2A> Dit bericht wordt rekening gehouden met de huidige systeemcultuur.       De volgende tabel toont de eerste waarden voor het nieuwe exemplaar.      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| De gelokaliseerde fouttekenreeks bericht. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public SignatureVerificationFailedException ();
    parameters: []
  overload: System.IdentityModel.SignatureVerificationFailedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.IdentityModel.SignatureVerificationFailedException
  langs:
  - csharp
  name: SignatureVerificationFailedException(String)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(String)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(String)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.IdentityModel.SignatureVerificationFailedException&quot;> </xref> klasse met een opgegeven foutbericht."
  remarks: "Deze constructor initialiseert de <xref:System.Exception.Message%2A>eigenschap van de nieuwe uitzondering via de `message` parameter.</xref:System.Exception.Message%2A> De inhoud van `message` is bedoeld om te worden ge√Ønterpreteerd door mensen. De aanroeper van deze constructor is vereist om ervoor te zorgen dat deze tekenreeks gelokaliseerd voor de huidige systeemcultuur.       De volgende tabel toont de eerste waarden voor het nieuwe exemplaar.      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| De fouttekenreeks bericht. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public SignatureVerificationFailedException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Het foutbericht met uitleg over de reden voor de uitzondering."
  overload: System.IdentityModel.SignatureVerificationFailedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.IdentityModel.SignatureVerificationFailedException
  langs:
  - csharp
  name: SignatureVerificationFailedException(SerializationInfo,StreamingContext)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(SerializationInfo,StreamingContext)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.IdentityModel.SignatureVerificationFailedException&quot;> </xref> klasse met de geserialiseerde gegevens."
  remarks: "Deze constructor is aangeroepen tijdens de deserialisatie op het object exception verzonden via een stroom opnieuw samenstellen."
  syntax:
    content: protected SignatureVerificationFailedException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Een <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> object dat de gegevens geserialiseerd object bevat."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Een <xref:System.Runtime.Serialization.StreamingContext>object dat de contextuele informatie over de bron- of doelserver bevat.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.IdentityModel.SignatureVerificationFailedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.IdentityModel.SignatureVerificationFailedException
  langs:
  - csharp
  name: SignatureVerificationFailedException(String,Exception)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(String,Exception)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(String,Exception)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.IdentityModel.SignatureVerificationFailedException&quot;> </xref> klasse met een opgegeven foutbericht en een verwijzing naar de interne uitzondering dat de oorzaak van deze uitzondering."
  remarks: "Een uitzondering die is gegenereerd als direct resultaat van een eerdere uitzondering moet een verwijzing bevatten naar de vorige uitzondering in de <xref:System.Exception.InnerException%2A>eigenschap.</xref:System.Exception.InnerException%2A> De <xref:System.Exception.InnerException%2A>eigenschap dezelfde waarde die is doorgegeven aan de constructor.</xref:System.Exception.InnerException%2A>       De volgende tabel toont de eerste waarden voor het nieuwe exemplaar.      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| De verwijzing naar de interne uitzondering. |   | <xref:System.Exception.Message%2A>| De fouttekenreeks bericht. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public SignatureVerificationFailedException (string message, Exception inner);
    parameters:
    - id: message
      type: System.String
      description: "Het foutbericht met uitleg over de reden voor de uitzondering."
    - id: inner
      type: System.Exception
      description: "De <xref:System.Exception>die de oorzaak van de huidige uitzondering.</xref:System.Exception> Als de `inner` parameter is niet <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, de huidige uitzondering is opgetreden tijdens een <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> blok die verantwoordelijk is voor de interne uitzondering."
  overload: System.IdentityModel.SignatureVerificationFailedException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.IdentityModel.Tokens.SecurityTokenException
  isExternal: false
  name: System.IdentityModel.Tokens.SecurityTokenException
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor
  parent: System.IdentityModel.SignatureVerificationFailedException
  isExternal: false
  name: SignatureVerificationFailedException()
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException()
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException()
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String)
  parent: System.IdentityModel.SignatureVerificationFailedException
  isExternal: false
  name: SignatureVerificationFailedException(String)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(String)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.IdentityModel.SignatureVerificationFailedException
  isExternal: false
  name: SignatureVerificationFailedException(SerializationInfo,StreamingContext)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(SerializationInfo,StreamingContext)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String,System.Exception)
  parent: System.IdentityModel.SignatureVerificationFailedException
  isExternal: false
  name: SignatureVerificationFailedException(String,Exception)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(String,Exception)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor*
  parent: System.IdentityModel.SignatureVerificationFailedException
  isExternal: false
  name: SignatureVerificationFailedException
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException
